update_service/installing-update-service.adoc

```diff
@@ -0,0 +1,58 @@
+[id="installing-update-service"]
+= Installing and configuring the OpenShift Update Service
+include::modules/common-attributes.adoc[]
+:context: update-service
+
+toc::[]
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
+
+You can install and configure the OpenShift Update Service to provide over-the-air updates for your cluster and its underlying operating system.
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+[id="update-service-prereqs"]
+== Prerequisites
+
+* For more information on installing Operators, see xref:../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
+
+[id="update-service-install"]
+== Installing the OpenShift Update Service Operator
+
+The OpenShift Update Service is installed by first installing the OpenShift Update Service Operator by using the {product-title} web console or CLI.
```

Comment:
For the clusters that are installed in a restricted network, OLM by default cannot access the OperatorHub sources. So the osus operator is not able to be installed directly if OLM is not customized in restricted network. I would suggest to have a note to explain it. For more information, see Using Operator Lifecycle Manager on restricted networks.

Reply:
Agree, done.

Reply:
[Using Operator Lifecycle Manager on restricted networks](https://docs.openshift.com/container-platform/4.6/operators/admin/olm-restricted-networks.html) is for mirroring catalog image in restricted network. But it does not mention the dedicated operator/container/metadata images mirror. So we'd better mention this step since it's requested for deploying osus in restricted network.

Reply:
Added new section `Configuring the Cluster Version Operator (CVO)`.

Reply:
> ...the dedicated operator/container/metadata images mirror...

Which components are these? We're covering the graph data and release images elsewhere so not sure what these are. It may be helpful too if you could show where they are covered in the developer-preview instructions here https://www.openshift.com/blog/openshift-update-service-update-manager-for-your-cluster.